Mathlib/GroupTheory/QuotientGroup.lean

```diff
@@ -132,7 +133,7 @@ theorem eq_iff_div_mem {N : Subgroup G} [nN : N.Normal] {x y : G} :
 
 @[to_additive]
 instance Quotient.commGroup {G : Type*} [CommGroup G] (N : Subgroup G) : CommGroup (G ⧸ N) :=
-  { toGroup := have := N.normal_of_comm; QuotientGroup.Quotient.group N
+  { toGroup := @QuotientGroup.Quotient.group _ _ N N.normal_of_comm
```

Comment:
Why is this? I think the original version is preferred, because the order of implicit arguments are implementation detail and can easily change.

Reply:
This change came from a bad git merge of master, will fix